common.py

```diff
@@ -177,6 +182,14 @@ def __init__(self,fname) :
                 self.nbins_gal_shear_read+=nbins
                 if tr.consider_tracer :
                     self.nbins_gal_shear+=nbins
+                if tr.include_m_bias:
+                    self.include_m_bias = True
+                    self.npar_mbias += self.nbins_gal_shear
```

Comment:
I think you should add +=nbins, not self.nbins_gal_shear here. The reason being that there could be more than one shear tracer.